.github/ISSUE_TEMPLATE/cherry-pick-request.yml

```diff
@@ -0,0 +1,158 @@
+---
+name: Cherry-Pick Request
+description: Request a cherry-pick to an AMP release.
+labels: ["Type: Release", "Cherry-pick: Beta", "Cherry-pick: Experimental", "Cherry-pick: LTS", "Cherry-pick: Stable"]
+title: "\U0001F338 Cherry-pick request for #ISSUE into #RELEASE"
+body:
+- type: markdown
+  id: header
+  attributes:
+    value: |
+      Thanks for filling out this cherry-pick request.
+      - See AMP's [release schedule](https://github.com/ampproject/amphtml/blob/main/docs/release-schedule.md) to learn about how releases work.
+      - See AMP's [release calendar](https://amp-release-calendar.appspot.com) for information about past releases (e.g. versions, dates, submission times, notes).
+      - See AMP's [code contribution docs](https://github.com/ampproject/amphtml/blob/main/docs/contributing-code.md#Cherry-picks) for an overview of the cherry-pick process.
```

Comment:
Suggestion to move these links to the bottom and keep the most important things up top.

Reply:
Do quotes i.e. `>` use less whitespace? If so can we use that?
- testing
- testing

> testing
> testing

Reply:
Tried it, the difference is negligible and the text looks grayed out. Leaving this as a bulleted list because all other issue templates use the same style for top-level instructions, and because the instruction text is dropped from the issue that's created.